src/Tribe/Main.php

```diff
@@ -6133,5 +6134,58 @@ public function register_plugin_autoload_paths( ) {
 
 			$this->get_autoloader_instance()->register_prefixes( $prefixes );
 		}
+
+		/**
+		 * Create a Filter Bar upsell tab.
+		 * 
+		 * @since TBD
+		 */
+		public function do_fbar_upsell_tab() {
+			// Bail if Filter Bar is already installed.
+			if ( class_exists( 'Tribe__Events__Filterbar__View' ) ) {
+				return;
+			}
+
+			tribe_assets(
+				self::instance(),
+				[
+					[
+						'tec-admin-filterbar-upsell',
+						'tec-admin-filterbar-upsell.css',
+					],
+				],
+				'admin_enqueue_scripts',
```

Comment:
We need to make sure this only enqueues on the correct page, so use `conditional` argument.

Reply:
conditional added